data/projectfooter.yml

```diff
@@ -33,77 +33,45 @@ links:
         url: https://wildfly.zulipchat.com/
 
 more_links:
-  - title: Find more Red Hat Middleware Community Projects
+  - title: Find more Community Projects used with WildFly
     subfolderitems:
-      - page: APIMan
-        url: https://www.apiman.io/
       - page: Arquillian
         url: http://arquillian.org
       - page: Ansible Collection for Wildfly
         url: https://github.com/ansible-middleware/wildfly
       - page: Byteman
         url: http://byteman.jboss.org/
-      - page: CapeDwarf
-        url: http://capedwarf.org/
-      - page: Chronicler
-        url: https://github.com/alechenninger/chronicler
-      - page: Darcy
-        url: https://github.com/darcy-framework
-      - page: Debezium
-        url: http://debezium.io/
-      - page: Drools
-        url: https://www.drools.org/
-      - page: GateIn
-        url: http://gatein.jboss.org/
-      - page: Hawkular
-        url: http://www.hawkular.org/
-      - page: Hawtio
-        url: http://hawt.io/
       - page: Hibernate
         url: http://hibernate.org/
       - page: Infinispan
         url: http://www.infinispan.org/
-      - page: CodeReady Studio
-        url: https://developers.redhat.com/products/codeready-studio/overview
-      - page: JBoss Forge
-        url: http://forge.jboss.org/
+      - page: IronJacamar
+        url: https://www.ironjacamar.org/
+      - page: Jackson
+        url: hhttps://github.com/FasterXML/jackson
       - page: JBoss Logging
         url: https://github.com/jboss-logging
       - page: JBoss Remoting
         url: http://jbossremoting.jboss.org/
-      - page: jBPM
-        url: https://www.jbpm.org/
-      - page: JSFUnit
-        url: https://jsfunit.jboss.org/
       - page: Keycloak
         url: https://www.keycloak.org/
       - page: Mod Cluster
         url: http://modcluster.io/
-      - page: ModeShape
-        url: http://modeshape.jboss.org/
-      - page: OptaPlanner
-        url: https://www.optaplanner.org/
+      - page: Narayana
+        url: https://www.narayana.io/
       - page: Quarkus
         url: https://quarkus.io/
       - page: RESTEasy
         url: http://resteasy.jboss.org/
```

Comment:
Some what of a note to myself, this should be resteasy.dev now. I'll wait to send something up after this is merged.